cves/CVE-2017-5650.yml

```diff
@@ -101,9 +106,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: No unit tests were involved in the fix for this vulnerability.
```

Comment:
I would elaborate more here. Did you find unit tests of the vulnerable code? If not, where did you look? If yes, why weren't they involved in the fix? What did the fix involve that didn't require changing existing unit tests/adding new ones?

Reply:
Elaborated on why no unit tests are necessary.